vizro-core/tests/unit/vizro/actions/test_filter_interaction_action.py

```diff
@@ -101,7 +101,7 @@ def target_scatter_filtered_continent(request, gapminder_2007, scatter_params):
     if country:
         data = data[data["country"].isin([country])]
 
-    return px.scatter(data, **scatter_params).update_layout(margin_t=24)
+    return px.scatter(data, **scatter_params).update_layout(margin_t=24, title_pad_l=0, title_pad_r=0, margin_l=24)
```

Comment:
@petar-qb @antonymilne  - can you guys remind me again why we need this everywhere in the unit tests for the actions? 😬 

If we add more overwrites to optimise the charts inside the dashboard, this might get lengthier as well. Not sure if there is a better way of doing this?

Reply:
For example, I am wondering whether we even need to compare the entire fig. The template and custom dashboard overwrites are covered in other unit tests as well. So wouldn't it suffice if we just compare` fig[data]` as this is the relevant part for the actions unit tests? Or would that strip off too much?

I am also fine leaving it as is and we can get back to it if it indeed becomes lengthier 😄

Reply:
Actions like filter/parameter/filter_interaction always return "called" figure objects (like `vm.Graph.__call__()`) . 

As we apply style changes to the `actual` vm.Graph inside the `__call__()` method, we have to apply the same changes to the `expected` vm.Graph so we can compare them properly.

Reply:
> Or would that strip off too much

I think so, especially since the parameters affect the "layout" of the figure.

Reply:
Okay! I'll leave it as is then

Reply:
Agree this is a bit annoying with tests but let's leave it like this for now since a change I make soon might simplify them anyway.